newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js

```diff
@@ -49,6 +50,7 @@ export default class LocalPreviewLauncher extends React.Component<
     networkPreviewPort: null,
     networkPreviewError: null,
     previewGamePath: null,
+    debugToolsOpen: true,
```

Comment:
Let's use the same name as the webContents and call this `devToolsOpen` - unless debug mean some other tools later but I don't think that's the case.

Reply:
renamed now :)